lib/fetch/util.js

```diff
@@ -286,7 +286,160 @@ function clonePolicyContainer () {
 // https://w3c.github.io/webappsec-referrer-policy/#determine-requests-referrer
 function determineRequestsReferrer (request) {
   // TODO
-  return 'no-referrer'
+  // 1. Let policy be request's referrer policy.
+  const policy = request.referrerPolicy
+
+  // Return no-referrer when empty or policy says so
+  if (policy === '' || policy === 'no-referrer') return 'no-referrer'
```

Comment:
```suggestion
  if (policy === '' || policy === 'no-referrer') {
    return 'no-referrer'
  }
```

Reply:
This is not part of spec?

Reply:
Not explicitly, but if there's no value or the `policy` explicitly says that there's `no-referrer`, there's no point in continuing executing the spec algorithm. Any suggestion on how to handle it? 🙂